client/src/ui/organisms/placement/index.tsx

```diff
@@ -209,73 +264,127 @@ export function PlacementInner({
-                    : ""
-                }`}
-                target="_blank"
-                rel="noreferrer"
-              >
-                <img
-                  src={`/pimg/${encodeURIComponent(image || "")}`}
-                  aria-hidden="true"
-                  alt=""
-                  width={imageWidth}
-                  height={imageHeight}
-                ></img>
-                <span>{copy}</span>
-              </a>
-              {cta && (
-                <a
-                  className="pong-cta"
-                  data-pong="pong->click"
-                  href={`/pong/click?code=${encodeURIComponent(click)}${
-                    pong?.fallback
-                      ? `&fallback=${encodeURIComponent(pong?.fallback?.view)}`
-                      : ""
-                  }`}
-                  target="_blank"
-                  rel="noreferrer"
-                >
-                  {cta}
-                </a>
-              )}
-              <a
-                href={pong?.fallback?.by || "/en-US/advertising"}
-                className="pong-note"
-                data-pong="pong->about"
-                target="_blank"
-                rel="noreferrer"
-              >
-                {pong?.fallback?.by ? "Ads by Carbon" : "Mozilla ads"}
-              </a>
-            </p>
+            {cta}
+          </a>
+        )}
+        <a
+          href="/en-US/advertising"
+          className="pong-note"
+          data-pong="pong->about"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Mozilla ads
+        </a>
+      </p>
 
-            <a
-              className="no-pong"
-              data-pong={user?.isSubscriber ? "pong->settings" : "pong->plus"}
-              href={
-                user?.isSubscriber
-                  ? "/en-US/plus/settings?ref=nope"
-                  : "/en-US/plus?ref=nope#subscribe"
-              }
-            >
-              Don't want to see ads?
-            </a>
-          </section>
-        </>
-      )}
-    </>
+      <a
+        className="no-pong"
+        data-pong={user?.isSubscriber ? "pong->settings" : "pong->plus"}
+        href={
+          user?.isSubscriber
+            ? "/en-US/plus/settings?ref=nope"
+            : "/en-US/plus?ref=nope#subscribe"
+        }
+      >
+        Don't want to see ads?
+      </a>
+    </section>
+  );
+}
+
+function RenderHpPlacement({
+  place,
+  extraClassNames = [],
+  click,
+  image,
+  imageWidth,
+  imageHeight,
+  style,
+}: PlacementRenderArgs) {
+  return (
+    <section
+      ref={place}
+      className={["place", ...extraClassNames].join(" ")}
+      style={style}
+    >
+      <a
+        className="pong"
+        data-pong="pong->click"
+        href={`/pong/click?code=${encodeURIComponent(click)}`}
+        target="_blank"
+        rel="noreferrer"
+      >
+        <img
+          src={`/pimg/${encodeURIComponent(image || "")}`}
+          aria-hidden="true"
+          alt=""
```

Comment:
Can we set the `copy` in Kevel as alt text, and include it here? That way we don't have to hide it from the accessibility tree with `aria-hidden`